site/content/docs/5.2/helpers/stacks.md

```diff
@@ -19,9 +19,9 @@ Use `.vstack` to create vertical layouts. Stacked items are full-width by defaul
 
```

Comment:
In this file, we can see in action in dark mode the red outline reset button. Not sure it is very accessible in terms of contrast (regular text AA). Maybe we could double check all outline buttons in dark mode.